composer.json

```diff
@@ -21,6 +21,7 @@
     "suggest": {
         "predis/predis": "Required to use the Redis storage engine",
         "thobbs/phpcassa": "Required to use the Cassandra storage engine",
-        "aws/aws-sdk-php": "Required to use the DynamoDB storage engine"
+        "aws/aws-sdk-php": "Required to use the DynamoDB storage engine",
+        "firebase/php-jwt": "Required to use JWT features"
```

Comment:
for v2.x I think we should just make this required instead of suggesting it, since it's core functionality (as opposed to an optional storage class)